include/constants.dat

```diff
@@ -92,7 +114,7 @@ DAT(header_value_page_write_update, "update")
 DAT(header_value_page_write_clear, "clear")
 DAT(header_value_payload_format_nometadata, "application/json;odata=nometadata")
 DAT(header_value_payload_format_fullmetadata, "application/json;odata=fullmetadata")
-DAT(header_value_storage_version, "2017-11-09")
+DAT(header_value_storage_version, "2018-11-09")
```

Comment:
>2018-11-09 [](start = 35, length = 10)

I thought that there shouldn't be any version change for ADLS features.

Reply:
`2018-11-09` is required by ADLS, it's clearly stated in our internal documents.

Reply:
Then it is possibly wise to review the breaking change in 2018-03-28 and 2018-11-09 to see if the SDK is broken by service version upgrade.

Reply:
SAS may be breaking, will confirm